api/all/src/main/java/io/opentelemetry/api/trace/SpanContext.java

```diff
@@ -30,7 +30,7 @@
    * @return the invalid {@code SpanContext}.
    */
   static SpanContext getInvalid() {
-    return ImmutableSpanContext.getInvalid();
+    return ImmutableSpanContext.INVALID;
```

Comment:
looks like this change snuck in here. I'm ok with it since it's not changing any APIs.

Reply:
I made it for consistency with accessing the variables directly as I did in this PR